test/gui/tst_syncing/test.feature

```diff
@@ -256,3 +256,34 @@ Feature: Syncing files
         When the user clicks on the activity tab
         And the user selects "Not Synced" tab in the activity
         Then the file "Folder1/a\\a.txt" should be blacklisted
+
+
+    Scenario Outline: Verify one empty folder with a length longer than the allowed limit will not be synced
+        Given user "Alice" has set up a client with default settings
+        And user "Alice" has created a folder "<foldername>" inside the sync folder
+        When user "Alice" creates a folder "<foldername>/<foldername>" inside the sync folder
+        And user "Alice" creates a folder "<foldername>/<foldername>/<foldername>" inside the sync folder
+        And user "Alice" creates a folder "<foldername>/<foldername>/<foldername>/<foldername>" inside the sync folder
+        And user "Alice" creates a folder "<foldername>/<foldername>/<foldername>/<foldername>/<foldername>" inside the sync folder
+        And the user waits for folder "<foldername>/<foldername>/<foldername>/<foldername>/<foldername>" to be synced
+        Then as "Alice" folder "<foldername>" should exist on the server
+        And as "Alice" folder "<foldername>/<foldername>" should exist on the server
+        And as "Alice" folder "<foldername>/<foldername>/<foldername>" should exist on the server
+        And as "Alice" folder "<foldername>/<foldername>/<foldername>/<foldername>" should exist on the server
+        And as "Alice" folder "<foldername>/<foldername>/<foldername>/<foldername>/<foldername>" should exist on the server
+        Examples:
+            | foldername                                                      |
+            | An empty folder which name is obviously more than 59 characters |
+
+    
+    Scenario: Invalid system names are synced in linux
```

Comment:
We can remove this scenario if we include it in the scenarios `Syncing a file to the server` and `Syncing a folder to the server`

Reply:
I think it is better this way.